.github/scripts/create-preview-links.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Based on:
+ * https://github.com/Automattic/themes/blob/a0c9b91f827f46ed60c502d41ef881b2f0552f03/.github/scripts/create-preview-links.js
+ */
+
+/*
+ * This function creates the URL where to download the built & zipped plugin.
+ *
+ * @param {object} context - The context of the event that triggered the action.
+ * @param {number} number - The PR number where the plugin changes are located.
+ * @returns {string} - The URL where to download the built & zipped plugin, as an artifact of the workflow.
+ */
+function createBlueprintUrl(context, number) {
+	const { repo, owner } = context;
+	const workflow = encodeURI('Playground Preview');
+	const artifact = 'gatherpress-pr';
+	const proxy = 'https://hub.carsten-bach.de/gatherpress/plugin-proxy.php';
+	// const proxy = 'https://gatherpress.org/playground-preview/plugin-proxy.php';
```

Comment:
Swap out `hub.carsten-bach.de...` against `gatherpress.org...`, 
as soon as `gatherpress.org/playground-preview/plugin-proxy.php` is running.

Reply:
Is part of https://github.com/GatherPress/gatherpress/pull/741